src/ctia/flows/autoload.clj

```diff
@@ -46,7 +46,11 @@
 (s/defn autoload-hooks! :- (s/eq nil)
   "Should retrieve a list of Hook classes from `project.cljs`.
   All these classes must implement the `Hook` java interface."
-  ([] (load-hooks! (:hooks-classes (p/read))))
-  ([profiles :- [s/Keyword]] (load-hooks! (:hooks-classes (try (p/read "project.clj" profiles)
-                                                (finally
-                                                  nil))))))
+  ([]
+   (load-hooks!
+    (:hooks-classes (p/read))))
+  ([profiles :- [s/Keyword]]
+   (load-hooks!
+    (:hooks-classes (try (p/read "project.clj" profiles)
+                         (finally
+                           nil))))))
```

Comment:
I realize that this was already here, but is it really wise to be reading configuration information from the build description file? I'm happy to do this in testing, but in production it is causing the conflation of the role of the build file.